HW01/HW1---Cars-dataset-Finalized.md

```diff
@@ -0,0 +1,107 @@
+---
+title: "HW1 - Cars dataset Finalized"
+author: "Han"
+date: "1/21/2021"
+output: 
+  html_document:
+    keep_md: true
+  theme: paper
+  toc: true
+  toc_float: true
+  code_folding: "hide"
+---
+
+
+## HW1 Assignment
+
+Dataset used: cars
+
+Details regarding dataset cars
+
+```r
+# Shows dataset
+cars
+```
+
+```
+##    speed dist
+## 1      4    2
+## 2      4   10
+## 3      7    4
+## 4      7   22
+## 5      8   16
+## 6      9   10
+## 7     10   18
+## 8     10   26
+## 9     10   34
+## 10    11   17
+## 11    11   28
+## 12    12   14
+## 13    12   20
+## 14    12   24
+## 15    12   28
+## 16    13   26
+## 17    13   34
+## 18    13   34
+## 19    13   46
+## 20    14   26
+## 21    14   36
+## 22    14   60
+## 23    14   80
+## 24    15   20
+## 25    15   26
+## 26    15   54
+## 27    16   32
+## 28    16   40
+## 29    17   32
+## 30    17   40
+## 31    17   50
+## 32    18   42
+## 33    18   56
+## 34    18   76
+## 35    18   84
+## 36    19   36
+## 37    19   46
+## 38    19   68
+## 39    20   32
+## 40    20   48
+## 41    20   52
+## 42    20   56
+## 43    20   64
+## 44    22   66
+## 45    23   54
+## 46    24   70
+## 47    24   92
+## 48    24   93
+## 49    24  120
+## 50    25   85
+```
+
+```r
+# Summary data of cars dataset
+summary(cars)
+```
+
+```
+##      speed           dist       
+##  Min.   : 4.0   Min.   :  2.00  
+##  1st Qu.:12.0   1st Qu.: 26.00  
+##  Median :15.0   Median : 36.00  
+##  Mean   :15.4   Mean   : 42.98  
+##  3rd Qu.:19.0   3rd Qu.: 56.00  
+##  Max.   :25.0   Max.   :120.00
+```
+
+```r
+# Correlation of distance and speed
+cor(cars$speed, cars$dist)
+```
+
+```
+## [1] 0.8068949
+```
+
+## Plots
```

Comment:
This graph isn't showing up in the knitted doc. Is it supposed to be an attached image or an r plot?

Reply:
Might be a bug in .md files? I had to display the rich diff below to get the image to display, but it's in a separate github file, I think. Other than that, nice job!!